src/muuntaja/config/common.js

```diff
@@ -16,3 +19,32 @@ export const validationOff = {
   noAdditionalProperties: false, // Do not allow additional properties in fields
   strict: false // If true, set all validationOptions to true
 };
+
+
+// if environment is test
+//   - returns the year that is given as options property 'year'
+// otherwise returns the current year in format YYYY as integer
+export function getYear(options) {
+  const environment = readEnvironmentVariable('NODE_ENV');
+
+  // only used for tests
+  if (environment === 'test') {
```

Comment:
Nice use of node_env environment variable. But make sure it is in use when Github action run tests.

Im not sure if this is needed if function is created bit differently

Reply:
Function in common.js was modified and variable node_env is no longer used.